themes/bootstrap3/js/facets.js

```diff
@@ -116,18 +116,289 @@ VuFind.register('facetList', function FacetList() {
   return { setup: setup, getContent: getContent, updateContent: updateContent };
 });
 
+/* --- Multi Facets Handling --- */
+VuFind.register('multiFacetsSelection', function multiFacetsSelection() {
+  let globalAddedParams = [];
+  let globalRemovedParams = [];
+  let initialRawParams = window.location.search.substring(1).split('&');
+  let initialFilteredParams = initialRawParams.filter(function isFilter(obj) {
+    return obj.startsWith(encodeURI('filter[]='));
+  });
+  let dateSelectorIds = [];
+  let isMultiFacetsSelectionActivated = false;
+  let callbackOnApply;
+  let callbackWhenDeactivated;
+  let defaultContext;
+
+  function stickApplyFiltersButtonAtTopWhenScrolling() {
+    let applyFilters = defaultContext.getElementsByClassName('apply-filters')[0];
+    let checkbox = defaultContext.getElementsByClassName('apply-filters-selection')[0];
+    window.onscroll = function fixButton() {
+      // To handle delayed loading elements changing the elements offset in the page
+      // We update the offset, depending on if we past the button or not
+      if (checkbox.getBoundingClientRect().bottom < 0) {
+        applyFilters.classList.add('fixed');
+      } else {
+        applyFilters.classList.remove('fixed');
+      }
```

Comment:
It looks like there is no `.fixed` class in the themes or provided by Bootstrap. [Bootstrap 5 has `.fixed-top`](https://getbootstrap.com/docs/5.3/helpers/position/#fixed-top) but since we have to introduce something to bootstrap3 anyway, can we use the [`position: sticky`](https://developer.mozilla.org/en-US/docs/Web/CSS/position) CSS feature. For example, the following style will present normally until they reach the top or bottom of the screen. This would prevent the need for toggling the class on and off, we can set it once via PHP or JS.

```CSS
position: sticky;
top: 20px;
bottom: 20px;	
```

[There are sticky utilities in Bootstrap 5](https://getbootstrap.com/docs/5.3/helpers/position/#sticky-top).

Reply:
Some leftovers from an earlier attempt
I removed the function which was not useful anymore